packages/jest-emotion/src/enzyme.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+import type { Options } from './serializer'
+import { createSerializer as createEmotionSerializer } from './serializer'
+import { createSerializer as createEnzymeSerializer } from 'enzyme-to-json'
+
+const enzymeSerializer = createEnzymeSerializer({})
+
+const tickle = (wrapper: *) => {
+  if (typeof wrapper.dive === 'function') {
+    wrapper.find('EmotionCssPropInternal').forEach(el => el.dive())
```

Comment:
Couldn't we have a serializer that calls this and then calls out to the printer so we don't have to call enzyme-to-json ourselves?(We'd also want a WeakSet to store which wrappers we've done the printing on so we don't do it infinitely)

Reply:
The issue is order, we want to execute this on the enzyme objects, but our serializer works on the JSON objects. The other options would either to be to have two serializers, such as an `init-emotion-enzyme` serializer as well as the standard `jest-emotion` one, so the user could setup the correct order of `['init-emotion-enzyme', 'enzyme-to-json', 'jest-emotion']`, or two export an initializer so that the get snapshot call would look something like `expect(initializer(component)).toMatchSnapshot()`. Both of these seemed awkward DX, this alongside the fact that we really do rely on `enzyme-to-json` led to the idea of simply tying them together here. It turns out that this makes it easier to setup, too, as neglecting to include both serializers would cause an error the way it was setup before. Open to other ideas, but this seemed to offer the right balance of DX and functionality.

Reply:
Okay, SGTM